modules/apps/object/object-rest-api/src/main/java/com/liferay/object/rest/manager/v1_0/ObjectRelationshipElementsParser.java

```diff
@@ -26,6 +26,8 @@
 
 	public String getClassName();
 
+	public long getCompanyId();
```

Comment:
Maybe this commit should be sorted before the previous one, as the companyId is used there

Reply:
Make sense! You are right